composer.json

```diff
@@ -19,10 +19,10 @@
 
     "require": {
         "php":                      ">=5.3.2",
-        "symfony/framework-bundle": ">=2.0.0",
-        "symfony/finder":           ">=2.0.0",
-        "symfony/validator":        ">=2.0.0",
-        "knplabs/knp-snappy":       "*"
+        "symfony/framework-bundle": "~2.3|~3.0",
```

Comment:
<2.3 is no longer supported by Symfony, I don't think bundles should keep support for it.